api/query/atoms.go

```diff
@@ -1003,6 +1023,27 @@ func (t *AbstractTestLabel) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// UnmarshalJSON for AbstractTestWebFeature attempts to interpret a query atom as
+// {"feature":<web_feature_string>}.
+func (t *AbstractTestWebFeature) UnmarshalJSON(b []byte) error {
```

Comment:
Missing tests for this function

Reply:
I missed the atoms_test.go file completely. Will do this.